diff --git a/node_modules/hexo-generator-baidu-sitemap/baidusitemap.ejs b/node_modules/hexo-generator-baidu-sitemap/baidusitemap.ejs
index ab347e6..06b6197 100755
--- a/node_modules/hexo-generator-baidu-sitemap/baidusitemap.ejs
+++ b/node_modules/hexo-generator-baidu-sitemap/baidusitemap.ejs
@@ -12,7 +12,7 @@
   if(post.categories){ -%>
   <url>
     <loc><%- encodeURI(url + post.path) %></loc>
-    <lastmod><%= post.updated.toDate().toISOString().replace(/T.*$/i, "") || post.date.toDate().toISOString().replace(/T.*$/i, "") %></lastmod>
+    <lastmod><%= (post.updated || post.date).toISOString().replace(/T.*$/i, "") %></lastmod>
   </url>
 <%}}) -%>
 </urlset> -%>
